cranelift/codegen/meta/src/isa/x86/legalize.rs

```diff
@@ -589,6 +590,16 @@ fn define_simd(shared: &mut SharedDefinitions, x86_instructions: &InstructionGro
         );
     }
 
+    // SIMD vselect; if BLEND* instructions are not available
```

Comment:
nit: we might want to note here that using `bitselect` works because the entire lane is filled with 1s or 0s